.github/workflows/build-and-release.yml

```diff
@@ -28,13 +34,59 @@ jobs:
       version: ${{ steps.export.outputs.version }}
       runner-size: ${{ steps.export.outputs.runner-size }}
       infra-config-repo: ${{ steps.export.outputs.infra-config-repo }}
-
+      environment: ${{ steps.env-var.outputs.environment }}
+      commit-hash: ${{ steps.env-var.outputs.commit-hash }}
+      author-name: ${{ steps.env-var.outputs.author-name }}
+      core-repo-reference: ${{ steps.env-var.outputs.core-repo-reference }}
+      config-repo-reference: ${{ steps.env-var.outputs.config-repo-reference }}
 
     steps:
+      - id: env-var
+        run: |
+          # prod env from release
+          if [[ "${{ github.event_name }}" == "release" ]]; then
+            export ENVIRONMENT=prod
+            echo "::set-output name=commit-hash::$GITHUB_SHA"
```

Comment:
Does the commit-hash var have to be echo'd out this way or can it be exported (as the other variables are) to keep it in-line with the other declarations?